unitTests/ui/OmniboxTest.ts

```diff
@@ -28,6 +31,21 @@ export function OmniboxTest() {
       Simulate.removeJQuery();
     });
 
+    function setupEnv() {
+      testEnv = new Mock.MockEnvironmentBuilder();
+      testEnv.searchInterface.getOmniboxAnalytics = jasmine.createSpy('omniboxAnalytics').and.returnValue(new OmniboxAnalytics()) as any;
+    }
+
+    function initOmnibox(options: IOmniboxOptions) {
+      setupEnv();
```

Comment:
This is where I had my test problem. 

I thought that since it was already initialize in the `beforeEach` in the root, I didn't need to reinit it here, but if i didn't,
all my test which check if the `simulation.queryBuilder.xxxxxxx` was set with what was in the option failed.

If you have any idea why i would appreciate it.

Reply:
Hmm, I'm not sure just yet, but I have some suggestions that might make the testing simpler. Let's discuss tomorrow on a call 👍